app/build.gradle

```diff
@@ -78,7 +78,7 @@ android {
         proguardFile '../config/proguard/retrolambda.pro'
         proguardFile '../config/proguard/rxjava.pro'
 
-        testInstrumentationRunner 'android.support.test.runner.AndroidJUnitRunner'
+        testInstrumentationRunner 'com.mycompany.myapp.TestButlerRunner'
```

Comment:
Can we call this something like `CustomAppTestRunner`? We'll have more than just Test Butler stuff in there.